examples/minimal-mdns/advertiser.cpp

```diff
@@ -120,10 +120,18 @@ bool HandleOptions(const char * aProgram, OptionSet * aOptions, int aIdentifier,
         gOptions.pairingHint = Optional<uint8_t>::Value(static_cast<uint8_t>(atoi(aValue)));
         return true;
     case kOptionOperationalFabricId:
-        gOptions.fabricId = atoll(aValue);
+        if (sscanf(aValue, "%" SCNx64, &gOptions.fabricId) != 1)
```

Comment:
SCNx64 or SCNu64?  That is, is the expected value hex or base-10?  The `atoll` we used to have was base-10, but maybe base-16 is in fact better?  If so, the commit message should explain the change.

Reply:
I think we should use hex because that's what the protocol uses in mDNS.

Reply:
That's fine, but (for future), it would be good if that were reflected in the commit message.  The commit message makes this sound like a pure build fix, but it includes a behavior change too....